src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -130,6 +130,10 @@ Frustum::Frustum(const std::array<std::array<double, 2>, 4>& face_vertices,
     w_plus_ = w_delta + 1.0;
     w_minus_ = w_delta - 1.0;
   } else {
+    zeta_distribution_value_ =
+        distribution_value.has_value()
+            ? distribution_value.value()
+            : -(radius_ + inner_radius_) / (radius_ - inner_radius_);
```

Comment:
This can be

```cpp
    zeta_distribution_value_ =
        distribution_value.value_or(
            -(radius_ + inner_radius_) / (radius_ - inner_radius_));
```